package.json

```diff
@@ -27,8 +27,8 @@
     "typecheck": "flow check",
     "build": "yarn run build:clean && yarn run build:lib && yarn run build:flow && yarn run build:dist",
     "build:clean": "rimraf lib && rimraf dist",
-    "build:dist": "rollup -c && rollup -c --min",
-    "build:lib": "babel src -d lib",
+    "build:dist": "rollup -c && cross-env NODE_ENV=min rollup -c",
```

Comment:
can you please explain your thinking behind this one?

Reply:
`--min` can be considered as rollup option. But it's not. This may confuse. Specifying env I think safer way to customize config.